libs/execution_context/ebpf_maps.c

```diff
@@ -19,77 +21,33 @@ typedef struct _ebpf_map_function_table
     void (*delete_map)(_In_ ebpf_core_map_t* map);
     uint8_t* (*find_entry)(_In_ ebpf_core_map_t* map, _In_ const uint8_t* key);
```

Comment:
Couldnt these have is_helper parameter and let the individual map implementations decide if they want to allow calls from helper functions or not?

Reply:
if the short term implementation becomes long term, yes.  If the short term is just short term, then it's better not to.  I can't tell yet :)